Dockerfile

```diff
@@ -115,7 +105,7 @@ COPY . ${ARCHES_ROOT}
 WORKDIR ${ARCHES_ROOT}
 
 RUN . ../ENV/bin/activate \
-    && pip install -e . --no-binary :all:
+    && pip install -e '.[dev]' --no-binary :all:
```

Comment:
We aren't committing to keeping this file up to date, but @aarongundel does this pass the smell test?

Reply:
I'm not familiar with the syntax/argument, so I'm not sure what you're trying to do here.

Reply:
I'm replacing requirements_dev.txt by declaring a group of optional dependencies in pyproject.toml. That group I'm calling "dev", and it can be given to pip with brackets. That gives either `arches[dev]` or locally `.[dev]`, but the latter fails at the shell level unless quoted.

Mostly I just wanted to know whether it's safe to remove the lines I removed just above this. I removed them because the requirements files went away, but I don't know what --no-binary and :all: are doing here.

Reply:
It is hard for me to say there, as we are not the main users of this dockerfile.  IMO this is probably ok - I don't see why this would break.